pkg/helm/helm.go

```diff
@@ -60,21 +49,20 @@ var (
 
 // Hydrator runs the helm hydration process.
 type Hydrator struct {
-	Chart                   string
-	Repo                    string
-	Version                 string
-	ReleaseName             string
-	Namespace               string
-	DeployNamespace         string
-	Values                  string
-	ValuesFileSources       []string
-	IncludeCRDs             string
-	HydrateRoot             string
-	Dest                    string
-	Auth                    configsync.AuthType
-	UserName                string
-	Password                string
-	ValuesFileApplyStrategy string
+	Chart             string
+	Repo              string
+	Version           string
+	ReleaseName       string
+	Namespace         string
+	DeployNamespace   string
+	Values            string
```

Comment:
Values is not a path, it is the literal yaml string with the inline values so it can't be appended or treated the same as ValuesFileSources without some larger implementation changes.